webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -1226,9 +1226,15 @@ function reloadTable(gridId) {
 function createNodeTable(gridId, nope, contextPath, refresh) {
 
 
-  columns = [ allColumns["Hostname"],  allColumns["OS"],  allColumns["Compliance"],  allColumns["Last run"]]
+  var cacheId = gridId + "_columns"
+  var cacheColumns = localStorage.getItem(cacheId)
```

Comment:
could you introduce a versionning in this feature ? If someone upgrade Rudder, or switch from one version to another, I see potential issues with fetched columns